sphinx/tutorial/rose/furthertopics/optional-configurations.rst

```diff
@@ -1,3 +1,5 @@
+.. _rose-turorial-optional-configurations:
```

Comment:
Typo (tu**r**orial) which is not insignificant because the reference will break on (the new) line 16 of `furthertopics/rose-bunch.rst` etc.